src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1464,6 +1464,10 @@ private function registerMessengerConfiguration(array $config, ContainerBuilder
 
         $messageToSenderIdsMapping = array();
         foreach ($config['routing'] as $message => $messageConfiguration) {
+            if (!class_exists($message) && !interface_exists($message)) {
```

Comment:
missing false as 2nd arg to interface_exists

Reply:
Why wouldn't I want the autoload for the interface? 🤔

Reply:
autoload has been triggered by the first check already, no need to do it twice. 
Actually both calls could take `false`, there is already an `interface_exists` call l1444 above

Reply:
> autoload has been triggered by the first check already, no need to do it twice.

that's the reason indeed

> there is already an interface_exists call l1444 above

but for another interface, so not sure what you mean :)

Reply:
nvm :) commented too fast

Reply:
Alright, very nice. Thanks for that, updated 👍